ee/hogai/eval/eval_root.py

```diff
@@ -455,6 +455,54 @@ async def eval_root(call_root, pytestconfig):
                     id="call_doc_search_32",
                 ),
             ),
+            # Ensure calls docs, not insights
```

Comment:
@tatoalo Could you clarify your branch question, please? The current answer - I didn't know we could do that 😄